scripts/vote_3_step_1[rpm]create_repository.sh

```diff
@@ -19,17 +27,19 @@ then
 fi
 echo
 
+
 #
 # Build package
 #
 echo "# Building RPM package #"
-if [ ! -f rpmbuild ]; then rm -rf rpmbuild; fi
-mkdir -pv rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+if [ -d rpmbuild ]; then rm -r rpmbuild; fi
+mkdir -pv rpmbuild/{BUILD,RPMS,SRPMS}
 cp -rfv git/packaging/rpm/* rpmbuild
-cp -rfv svn/vote/apache-ignite-fabric-${ignite_version}-bin.zip rpmbuild/SOURCES/apache-ignite.zip
+cp -rfv svn/vote/apache-ignite-fabric-${ignite_version}-bin.zip rpmbuild/SOURCES/
 rpmbuild -bb --define "_topdir $(pwd)/rpmbuild" rpmbuild/SPECS/apache-ignite.spec
```

Comment:
I thought we don't want to be a 'fabric' anymore.

Reply:
Not until https://issues.apache.org/jira/browse/IGNITE-7251 is reviewed and merged to master.